pkg/terraform/exec/plugins/vendor/github.com/terraform-providers/terraform-provider-azurerm/azurerm/resource_arm_loadbalancer.go

```diff
@@ -68,10 +68,9 @@ func resourceArmLoadBalancer() *schema.Resource {
 						},
 
 						"private_ip_address": {
-							Type:         schema.TypeString,
-							Optional:     true,
-							Computed:     true,
-							ValidateFunc: validate.IPv4AddressOrEmpty,
+							Type:     schema.TypeString,
+							Optional: true,
+							Computed: true,
```

Comment:
I assume there should still be a `ValidateFunc`; you don't want it to accept random strings

Reply:
I would have thought so as well, however, when poking through the terraform source, there is no validation for all the other private_ip_address fields scattered around the source.